src/components/ModelRun.vue

```diff
@@ -514,7 +498,7 @@ export default {
   },
   watch: {
```

Comment:
I think this whole watch block probably needs to go - it's riskier now - I don't see this.value is used anywhere, but if it was in the future, we could end up with a weird bug where the current model's data gets overwritten with it. Let's talk about what you're looking to do with this block at 3pm